star_profile.cpp

```diff
@@ -127,20 +151,20 @@ void ProfileWindow::OnPaint(wxPaintEvent& WXUNUSED(evt))
 
     int i;
     int *profptr;
-    wxString label;
+    string profileLabel;
     switch (this->mode) {  // Figure which profile to use
     case 0: // mid-row
     default:
         profptr = midrow_profile;
-        label = _("Mid row");
+        profileLabel = "Mid row";

```

Comment:
I just pushed a correction to this.
Hallgeir

2015-10-29 16:39 GMT+01:00 Andy Galasso <notifications@github.com>:

> In star_profile.cpp
> <https://github.com/OpenPHDGuiding/phd2/pull/477#discussion_r43401675>:
>
> >      switch (this->mode) {  // Figure which profile to use
> >      case 0: // mid-row
> >      default:
> >          profptr = midrow_profile;
> > -        label = _("Mid row");
> > +        profileLabel = "Mid row";
>
> _("Mid row") is important since this marks the string for translation
> (gettext). We need to keep the _() !
> _() is a macro that expands to wxGetTranslation() so if the user has
> selected a different language, "Mid row" is replaced by the corresponding
> localized version of the text.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenPHDGuiding/phd2/pull/477/files#r43401675>.
>